### ISSUE TITLE
How to enable Latex in this theme

### ISSUE THREAD

**Issue by hatute** [klugjo/hexo-theme-clean-blog#60]
Since I need to post some articles with latex equations, and install the Hexo Math plugin. But when the site deployed, the latex part still cannot display properly.
Should I add some configurations in the theme's `_config.yml` or something else ?

**Comment by klugjo** [comment]
Hi,

It is not supported at this time. Do you know a good library to display latex equations on a page, or do you know another theme that has it ?

**Comment by hatute** [comment]
@klugjo 
Hi,

For now, I found two ways to solve this problem.
1. Add a self-made file `mathjax.ejs` under the `clean-blog/layout/`
```
<!-- mathjax config similar to math.stackexchange -->

<script type="text/x-mathjax-config">
    MathJax.Hub.Config({
      tex2jax: {
        inlineMath: [ ['$','$'], ["\\(","\\)"] ],
        processEscapes: true
      }
    });
  </script>
  
  <script type="text/x-mathjax-config">
      MathJax.Hub.Config({
        tex2jax: {
          skipTags: ['script', 'noscript', 'style', 'textarea', 'pre', 'code']
        }
      });
  </script>
  
  <script type="text/x-mathjax-config">
      MathJax.Hub.Queue(function() {
          var all = MathJax.Hub.getAllJax(), i;
          for(i=0; i < all.length; i += 1) {
              all[i].SourceElement().parentNode.className += ' has-jax';
          }
      });
  </script>
  
  <script src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML' async></script> 
```
and then add `<%- partial('mathjax') %>` between `<body> ... </body>` label

2. Change to [hexo-theme-next](https://github.com/theme-next/hexo-theme-next) which supports latex

**Comment by stevenjoezhang** [comment]
You can use this Hexo plugin: https://github.com/next-theme/hexo-filter-mathjax
